README.md

```diff
@@ -35,5 +35,12 @@ A Persian (Farsi) Font
            url('Samim-Bold.ttf') format('truetype');
       font-weight: bold;
     }
+    
+    body {
+        /* optional enhancements for better rendering */
+        text-rendering: auto;
+        -webkit-font-smoothing: antialiased;
+        -moz-osx-font-smoothing: grayscale;
```

Comment:
It's a bit selfish of mine to ask for removal of this. I was looking at some of the issues on Vazir font with some screenshots which are horrible in resolution. 
Normal people don't have high end devices like us! I keep forgetting this!